### PR TITLE
[DEVREL-588] Replace jQuery with Vanilla JS in Demo & Play Along apps

### PR DESCRIPTION
**Pull request by ravirajsubramanian**

### CODE REVIEW

File: Play-Along/Exercises/fundamentals/level2/app/modal/modal.html [hunk 1]
```diff
@@ -1,7 +1,6 @@
 <html>
 
 <head>
-  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.0/jquery.min.js"></script>
   <script src="https://static.freshdev.io/fdk/2.0/assets/fresh_client.js"></script>
   <script src="modal.js"></script>
   <link rel="stylesheet" type="text/css" href="https://static.freshdev.io/fdk/2.0/assets/freshdesk.css">
```
Comment (Saif-Shines) [inline Play-Along/Exercises/fundamentals/level2/app/modal/modal.html:6]: I see this HTML file doesn't use `freshdesk.css` stylesheet at all. It appears to be redundant. If you don't see it's being used, then I suppose it's safe to be removed.

File: Play-Along/Exercises/fundamentals/level2/start/app/modal/modal.html [hunk 2]
```diff
@@ -1,7 +1,6 @@
 <html>
 
 <head>
-  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.0/jquery.min.js"></script>
   <script src="https://static.freshdev.io/fdk/2.0/assets/fresh_client.js"></script>
   <script src="modal.js"></script>
   <link rel="stylesheet" type="text/css" href="https://static.freshdev.io/fdk/2.0/assets/freshdesk.css">
```
Comment (Saif-Shines) [inline Play-Along/Exercises/fundamentals/level2/start/app/modal/modal.html:6]: Let's have this removed, if the HTML doesn't use this stylesheet.

File: Play-Along/Tutorials/sample_full_page_app/app/app.js [hunk 3]
```diff
@@ -72,22 +72,23 @@ $(document).ready( () => {
...
-        $('.status').removeClass('active');
-        $(this).addClass('active');
-        const tickets = data[$(this).data('index')].results;
-        
-        $('#ticket-list').html(`
+      document.querySelector('.status').addEventListener('click', function () {
+        document.getElementById('chart').classList.add('minimize');
+        document.querySelector('.status').classList.remove('active');
+        this.classList.add('active');
+        const tickets = data[this.dataset.index].results;
```
Comment (Saif-Shines) [inline Play-Along/Tutorials/sample_full_page_app/app/app.js:85]: This HTML5 dataset feature is so good! Nicely leveraged.

**Review by Saif-Shines — APPROVED** [review]
Done reviewing. Decision to improve on Tutorials apps first is good choice.

**Review by velmurugan-balasubramanian — APPROVED** [review]

File: Play-Along/Exercises/fundamentals/level2/app/modal/modal.html (hunk 1 above)
Comment (ravirajsubramanian) [reply Play-Along/Exercises/fundamentals/level2/app/modal/modal.html:6]: I wanted the changes to be lean which is why I didn't check for other changes. Maybe I will do this in the further cleanup.

File: Play-Along/Exercises/fundamentals/level2/start/app/modal/modal.html (hunk 2 above)
Comment (ravirajsubramanian) [reply Play-Along/Exercises/fundamentals/level2/start/app/modal/modal.html:6]: Same as before.

**Pull request merged**